bootstrap/cmd/kfctl/cmd/apply.go

```diff
@@ -72,15 +52,35 @@ var applyCmd = &cobra.Command{
 			if err != nil || kfApp == nil {
 				return fmt.Errorf("error loading kfapp: %v", err)
 			}
+			return kfApp.Apply(kftypes.ALL)
```

Comment:
I think we can get rid of the call to GetKfApplyFromCFGFile and kfApp.Apply here.
In the if block we just need to populate configFilePath because its not suplied via command line argument.

At that point the code path should be the same as when user supplies -f flag.

Reply:
I think we can also delete GetKfAppFromCfgFile and just use BuildKfAppFromURI call below.

Reply:
We will need to update delete.go to call LoadKfAppFromCfgFile as well rather than GetKfApplyFromCFGFile